### DISCUSSION TITLE
Add bindable shortcuts to show/hide any app that's open

### DISCUSSION THREAD

**Discussion by shamilovtim** [rxhanson/Rectangle#710] [Ideas]
Depending on appropriateness for this application. Inspired by `ctrl ~` for iTerm I'd like to allow users to select any valid app binary and when that app is open the programmed shortcut will show/hide that application. 

I was planning on making this my own standalone Mac app but I don't really have much experience (or desire) to bootstrap and ship an entire MacOS project so maybe I could just help add it to Rectangle instead

**Comment by rxhanson** [comment]
Thanks for looking to contribute! Sorry for the delayed reply.

In an effort to keep Rectangle really slim, I think my preference would be to keep this outside of Rectangle. You could always fork Rectangle and if it's a popular item then I could see merging it in down the road. 

There's a myriad of launchers out there that include this type of functionality. I can't remember the name of the free one on the mac app store at the moment. The difference there is in hiding, but with cmd+h being hide, I think the value is in launching specific apps. My personal launcher that I built is Charmstone, https://charmstone.app. You can check a box to let it bind your shortcut with arrow keys to show it.
